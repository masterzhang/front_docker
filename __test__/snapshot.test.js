import React from 'react'
import TestRenderer from 'react-test-renderer'
import intl from '@gem-mine/intl-react'
import { ReactRouterDom } from '@gem-mine/durex-router'
import BasicRouter from '@/page/home/Dashboard'

test('Error renderer', async (done) => {
  await intl.init({
    locale: 'zz',
    locales: {
      zz: {}
    }
  })
  const renderer = TestRenderer.create(
    <ReactRouterDom.BrowserRouter>
      <BasicRouter />
    </ReactRouterDom.BrowserRouter>
  )
  expect(renderer.toJSON()).toMatchSnapshot()
  done()
})
