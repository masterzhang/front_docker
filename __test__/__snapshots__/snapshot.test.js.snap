// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error renderer 1`] = `
<div
  className="dashboard"
>
  <div
    className="logo"
  />
  <div
    className="welcome"
  >
    <span
      className="name"
    >
      Fish Pro
    </span>
    <span>
      一个更好的后台管理系统
    </span>
  </div>
  <div
    className="team"
  >
    WEB 端公共能力团队
  </div>
</div>
`;
